### ISSUE TITLE
`parse()` privatekey  Isn't fully implemented.

### ISSUE THREAD

**Issue by aleksiy325** [dotenvx/dotenvx#472]
Looks like the parse function in the `lib/main.js`  isn't fully implemented. Theres a todo

https://github.com/dotenvx/dotenvx/blob/2aa660695757143f65751a201115f074b81942a8/src/lib/main.js#L156-L157

I would like to read the config from a string instead of a file path.  I've work around it temporarily by mocking out `fs` but would be nice to have it done the proper.

```ts
import { config, DotenvParseOutput } from "@dotenvx/dotenvx";
import mock from "mock-fs";

const path = "/.app-config/.env";
mock({
    [path]: raw,
});
const loadedConfig = config({ path: path, override: true });
mock.restore();
``` 

I guess getting the `privateKey` like in `Run` would be the solution? 
https://github.com/dotenvx/dotenvx/blob/2aa660695757143f65751a201115f074b81942a8/src/lib/services/run.js#L95-L97

Thanks!

**Comment by motdotla** [comment]
effort started here https://github.com/dotenvx/dotenvx/pull/474

**Comment by motdotla** [comment]
thanks for surfacing. now available in `1.26.0` or greater

<img width="1898" alt="image" src="https://github.com/user-attachments/assets/4bae5ca3-9ea0-4e4b-91ed-d4226acf0e88">

**Comment by aleksiy325** [comment]
Awesome, really appreciate it!
